airbyte-commons-worker/build.gradle

```diff
@@ -0,0 +1,65 @@
+import org.jsonschema2pojo.SourceType
+
+plugins {
+    id "java-library"
+    id 'com.github.eirnym.js2p' version '1.0'
+}
+
+group 'io.airbyte...'
+version '0.40.14'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    annotationProcessor platform(libs.micronaut.bom)
+    annotationProcessor libs.bundles.micronaut.annotation.processor
+
+    implementation platform(libs.micronaut.bom)
+    implementation libs.bundles.micronaut
+
+    implementation 'io.fabric8:kubernetes-client:5.12.2'
+    implementation 'io.temporal:temporal-sdk:1.8.1'
+    implementation 'org.apache.ant:ant:1.10.10'
+    implementation 'org.apache.commons:commons-text:1.9'
+
+    implementation project(':airbyte-commons-protocol')
+    implementation project(':airbyte-commons-temporal')
+    implementation project(':airbyte-config:config-models')
+    implementation project(':airbyte-json-validation')
+    implementation project(':airbyte-metrics:metrics-lib')
+    implementation project(':airbyte-persistence:job-persistence')
+    implementation project(':airbyte-protocol:protocol-models')
+
+    testAnnotationProcessor platform(libs.micronaut.bom)
+    testAnnotationProcessor libs.bundles.micronaut.test.annotation.processor
+
+    testImplementation libs.bundles.micronaut.test
+    testImplementation 'com.jayway.jsonpath:json-path:2.7.0'
+    testImplementation 'org.mockito:mockito-inline:4.7.0'
+    testImplementation libs.postgresql
+    testImplementation libs.platform.testcontainers
+    testImplementation libs.platform.testcontainers.postgresql
+
+    testImplementation project(':airbyte-commons-docker')
+}
+
+jsonSchema2Pojo {
+    sourceType = SourceType.YAMLSCHEMA
+    source = files("${sourceSets.main.output.resourcesDir}/workers_models")
+    targetDirectory = new File(project.buildDir, 'generated/src/gen/java/')
+    removeOldOutput = true
+
+    targetPackage = 'io.airbyte.persistence.job.models'
+
+    useLongIntegers = true
+    generateBuilders = true
+    includeConstructors = false
+    includeSetters = true
+}
+
+Properties env = new Properties()
+rootProject.file('.env.dev').withInputStream { env.load(it) }
```

Comment:
These two lines can be removed.  They are only necessary if you need to load environment variables from the .env file for use in a task (e.g. run, tests, etc).  They don't appear to be used anywhere in this project.

Reply:
Ok, I had issue with `S3DocumentStoreClientTest` which seemed to miss an env variable. Will retry without.